### DISCUSSION TITLE
Force monitor resolution to 1920 x 1080 pixels

### DISCUSSION THREAD

**Discussion by victormota-al** [guysoft/FullPageOS#603] [General]
Sometimes when I start the system the resolution is good, but most of the time the resolution is bad, a low resolution.

I'm using a touchscreen monitor + RPi4.
FullPageOS Nightly

I would like to know how I can force the resolution on startup?

**Comment by guysoft** [comment]
See the FAQ:
https://github.com/guysoft/FullPageOS/wiki/FAQ#FHD-on-4K-Display

Also was posted 2 weeks ago here. make sure to search first if you have a question :)
https://github.com/guysoft/FullPageOS/issues/601
